js/content/community.js

```diff
@@ -52,7 +26,9 @@ let ProfileData = (function(){
     let _promise = null;
     self.promise = async function() {
         if (!_promise) {
-            _promise = Background.action("profile", SteamId.getSteamId())
+            let steamId = SteamId.fromDOM();
+
+            _promise = Background.action('profile', { 'profile': steamId, } )
```

Comment:
This has still to be fixed
```suggestion
            _promise = Background.action("profile", SteamId.getSteamId())
```